lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -179,7 +179,8 @@ def populate_nonces(directive, source_list)
     # unsafe-inline, this is more concise.
     def append_nonce(source_list, nonce)
       if nonce
-        source_list.push("'nonce-#{nonce}'", UNSAFE_INLINE)
+        source_list.push("'nonce-#{nonce}'")
+        source_list.push(UNSAFE_INLINE) unless @config[:disable_append_unsafe_inline]
```

Comment:
Let's call this `disable_nonce_backwards_compatibility` since `append` is somewhat overloaded in this library.

Reply:
Sure, good idea. I forced pushed the previous commit with this renamed.